src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -82,11 +91,53 @@ object WdlExpression {
     val parseTree = parser.parse_e(tokens, new WdlSyntaxErrorFormatter(terminalMap))
     new WdlExpression(parseTree.toAst)
   }
+
+  def toString(ast: AstNode, highlighter: SyntaxHighlighter = NullSyntaxHighlighter): String = {
```

Comment:
Maybe we need @mcovarr as a tie breaker... my closing argument is that it looks fine as it is now and it's marginally more flexible and it's explicit and I don't feel that it's worth removing capabilities.  The code is pretty foolproof as it is now, I have a hard time seeing any risk in which "make the SyntaxHighlighter a case object that's defined based on what mode the user is running in" is the solution to our problem.